apiserver/pkg/model/converter.go

```diff
@@ -156,12 +146,19 @@ func FromCrdToApiJob(job *v1alpha1.RayJob) (pbJob *api.RayJob) {
 		ShutdownAfterJobFinishes: job.Spec.ShutdownAfterJobFinishes,
 		ClusterSelector:          job.Spec.ClusterSelector,
 		ClusterSpec:              PopulateRayClusterSpec(job.Spec.RayClusterSpec),
-		TtlSecondsAfterFinished:  ttl,
 		CreatedAt:                &timestamp.Timestamp{Seconds: job.CreationTimestamp.Unix()},
-		DeleteAt:                 &timestamp.Timestamp{Seconds: deleteTime},
 		JobStatus:                string(job.Status.JobStatus),
 		JobDeploymentStatus:      string(job.Status.JobDeploymentStatus),
 		Message:                  job.Status.Message,
 	}
+
+	if job.Spec.TTLSecondsAfterFinished != nil {
```

Comment:
Just like to confirm that this is just improvement, there's no bug or nil issue, right?

Reply:
yes